src/elements/tabs/tab-nav-bar/tab-nav-bar.global.scss

```diff
@@ -0,0 +1,16 @@
+@use '../../core/styles' as sbb;
+
+$theme: 'standard' !default;
+
+@mixin base {
+  // TODO: these variables are shared with the tab-label component. They should be merged once the css refactoring happen.
+  --sbb-tab-label-height: var(--sbb-size-element-m);
```

Comment:
Here should the theme pattern be applied and size s should be the default for lean. also for inline padding and font-size